src/darker/tests/test_git.py

```diff
@@ -449,3 +449,18 @@ def test_edited_linenums_differ_revision_vs_lines(git_repo, context_lines, expec
     result = differ.revision_vs_lines(Path("a.py"), content, context_lines)
 
     assert result == expect
+
+
+def test_local_gitconfig_ignored_by_gitrepofixture(tmp_path):
+    """Tests that ~/.gitconfig is ignored when running darker's git tests"""
+    (tmp_path / "HEAD").write_text("ref: refs/heads/main")
+
+    with patch.dict(os.environ, {"HOME": str(tmp_path)}):
+        # Note: once we decide to drop support for git < 2.28, the HEAD file
+        # creation above can be removed, and setup can simplify to
+        # check_call("git config --global init.defaultBranch main".split())
+        check_call("git config --global init.templateDir".split() + [tmp_path])
```

Comment:
It seems this needs to be turned into a string – py36 and py37 Windows builds are failing.
```suggestion
        check_call("git config --global init.templateDir".split() + [str(tmp_path)])
```

Reply:
Thanks for this!